_data/sites.yml

```diff
@@ -963,6 +963,11 @@
   size: 69.5
   last_checked: 2022-05-06
 
+- domain: daudix.codeberg.page
+  url: https://daudix.codeberg.page/
+  size: 40.8
+  last_checked: 2024-01-28 (YYYY-MM-DD)
```

Comment:
The placeholder shouldn't be there. 😅 Do you mind removing it?

Reply:
Oops, totally missed it 😅